src/db.js

```diff
@@ -122,10 +124,18 @@ const recordTransfer = (address) => {
   // record player address if not already
   const owners = currentPlayers();
   // if already played this round, ignore
-  if (owners.includes(address)) {
+  // if (owners.includes(address)) {
+  //   return;
+  // }
+  if (owners.filter((owner) => owner.address === address).length > 0) {
     return;
   }
-  owners.push(address);
+  // owners.push(address);
+  owners.push({
+    address,
+    timestamp: Date.now(),
```

Comment:
let's use `new Date().toISOString()` so we have the same format as the `lastTransferTime` in the game state